### PR TITLE
update duplicate css message

### PR DESCRIPTION
**Pull request by jakemac53**

### CODE REVIEW

**Comment by jakemac53** [comment]
fixes https://github.com/dart-lang/polymer-dart/issues/19

File: lib/src/build/messages.dart [hunk 1]
```diff
@@ -547,15 +547,52 @@ const CSS_FILE_INLINED_MULTIPLE_TIMES = const MessageTemplate(
...
+    Becomes:
+
+        <link rel="import" href="my_theme.html">
+        <polymer-element name="my-element">
+          <template>
+            <core-style ref="my-theme"></core-style>
+          </template>
+        </polymer-element>
+
+2. The alternative option is to use the following pattern to update your
```
Comment (sigmundch) [inline lib/src/build/messages.dart]: I'd put this first (it is a self contained solution that requires no extra dependencies)

File: lib/src/build/messages.dart [hunk 2]
```diff
@@ -547,15 +547,52 @@ const CSS_FILE_INLINED_MULTIPLE_TIMES = const MessageTemplate(
...
+    pubspec.yaml so this file is not inlined at all:
+
+        transformers:
+        - polymer:
+            inline_stylesheets:
+              web/my_file.css: false
+
+    If you would like to hide this warning and keep it inlined, do the same thing
```
Comment (sigmundch) [inline lib/src/build/messages.dart]: lets divide this in a separate one.

1. opt-out inlining in your `pubspec.yaml` 
2. opt-in for inlining in your `pubspec.yaml`
3. use `core-style` from `core_elements`

File: lib/src/build/messages.dart [hunk 3]
```diff
@@ -547,15 +547,52 @@ const CSS_FILE_INLINED_MULTIPLE_TIMES = const MessageTemplate(
     '''
 Css files are inlined by default, but if you import the same one in multiple
 places you probably want to override this behavior to prevent duplicate code.
-To do this, use the following pattern to update your pubspec.yaml:
 
-    transformers:
-    - polymer:
-        inline_stylesheets:
-          web/my_file.css: false
+There are two typical options for dealing with this:
```
Comment (sigmundch) [inline lib/src/build/messages.dart]: (two -> three, if we do what I mentioned below)

File: lib/src/build/messages.dart [hunk 4]
```diff
@@ -547,15 +547,52 @@ const CSS_FILE_INLINED_MULTIPLE_TIMES = const MessageTemplate(
...
 
-If you would like to hide this warning and keep it inlined, do the same thing
-but assign the value to true.
+1. Use the `core-style` element from the `core_elements` package.
+
+    The easiest way to do this is change your `*.css` file into a `*.html` file,
+    andwrap the entire thing in a `core-style` with an id, something like the
```
Comment (sigmundch) [inline lib/src/build/messages.dart]: andwrap => and wrap

File: lib/src/build/messages.dart (hunk 1 above)
Comment (jakemac53) [inline lib/src/build/messages.dart]: Its a strictly worse solution though.

File: lib/src/build/messages.dart (hunk 2 above)
Comment (jakemac53) [inline lib/src/build/messages.dart]: Ok will do, but I am going to put `core-style` first as it is the best solution ;)

File: lib/src/build/messages.dart (hunk 3 above)
Comment (jakemac53) [inline lib/src/build/messages.dart]: done

File: lib/src/build/messages.dart (hunk 4 above)
Comment (jakemac53) [inline lib/src/build/messages.dart]: done

**Comment by sigmundch** [comment]
lgtm, although the *recommended* *acceptable* *worst* labels might be a bit too much for me... how about we get rid of those, but keep the warnings you already have?

**Pull request merged**